python/rtclient/__init__.py

```diff
@@ -138,10 +138,10 @@ def __init__(
     def _user_agent(self):
         return "ms-rtclient-0.4.3"
 
-    def _get_auth(self):
+    async def _get_auth(self):
         if self._token_credential:
             scopes = ["https://cognitiveservices.azure.com/.default"]
-            token = self._token_credential.get_token(scopes)
+            token = await self._token_credential.get_token(scopes)
```

Comment:
Should you not leverage the async credentials as well, which you need to import from [azure.identity.aio](https://aka.ms/azsdk/python/identity/aio/docs)? Not sure if the current proposal will work in a non blocking way.

See https://learn.microsoft.com/en-us/python/api/overview/azure/identity-readme?view=azure-python#async-credentials